new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/Kubernetes.java

```diff
@@ -233,18 +275,24 @@ public static String getPodLog(String name, String namespace) throws ApiExceptio
    */
   public static String getPodLog(String name, String namespace, String container)
       throws ApiException {
-    String log = coreV1Api.readNamespacedPodLog(
-        name, // name of the Pod
-        namespace, // name of the Namespace
-        container, // container for which to stream logs
-        null, //  true/false Follow the log stream of the pod
-        null, // number of bytes to read from the server before terminating the log output
-        PRETTY, // pretty print output
-        null, // true/false, Return previous terminated container logs
-        null, // relative time (seconds) before the current time from which to show logs
-        null, // number of lines from the end of the logs to show
-        null // true/false, add timestamp at the beginning of every line of log output
-    );
+    String log = null;
+    try {
+      log = coreV1Api.readNamespacedPodLog(
```

Comment:
Just curious, why are we using GenericKubernetesApi for everything except Pod?

Reply:
Actually, looks like we're using both kinds of API's.  It doesn't really matter.